proxy/Dockerfile-deps

```diff
@@ -1,3 +1,10 @@
+# WARNING: because whether or not to build a new deps image is based on 
```

Comment:
mind mentioning https://github.com/runconduit/conduit/issues/115 ?

Reply:
@siggy done in https://github.com/runconduit/conduit/pull/116/commits/f3ae8a7fabf73722aa81e50b5458e4e18fe5e272